cmd/backend.go

```diff
@@ -34,13 +34,19 @@ func main() {
 		log.Info().Msgf("Successfully loaded environment variables from %s.", *envFile)
 	}
 
-	mig_err := utils.MigrateModels()
+	db, db_err := utils.GetDB()
+
```

Comment:
seems a `defer db.Close()` is missing here?

Reply:
weird, the connection pool should also have a close method. nevermind, seems fine then, though would need to test on multiple connections server once.